### ISSUE TITLE
Bundle on friday night

### ISSUE THREAD

**Issue by drapergeek** [railsbridge-boston/railsbridge-boston#62]
We should have the students run a `bundle install` on friday night instead of waiting until saturday halfway through the day to do this. We are already having them do a lot of things that require waiting on Friday night and this process took quite some time to complete on satuday.

**Comment by slothbear** [comment]
There are some comments on this in #18.

**Comment by danchoi** [comment]
See the proposal in #86

**Comment by decklin** [comment]
closing in favor of https://github.com/railsbridge-boston/railsbridge-vm/issues/8 (#86 was closed).
